update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -61,6 +61,7 @@ public static void main( String[] args ) throws IOException {
       dbaGkCentral = new MySQLAdaptor(hostGK, databaseGK, userGK, passwordGK, portGK);
     } catch (Exception e) {
       e.printStackTrace();
+      System.exit(0);
```

Comment:
For an irregular program termination, a non-zero System exit might be better.  `System.exit(1)` is okay here or perhaps even just remove the try block and add the exception to the main method's `throws` list.

Reply:
That's true, but the effect should be the same by adding to the `throws` list in terms of an irregular exit and a stack trace being printed.

Reply:
If the errors are not recoverable, then it's probably OK to just let them get thrown from `main`. On the other hand, in this case catching them and calling `System.exit(1)` is probably OK too, but there should be a more descriptive message, perhaps.

(In general, I don't like catching `Exception` - it's too broad. More specific exception types should be handled separately, when possible, but I know that many of the core Reactome libraries only throw `Exception` so there's not much we can do about that).

Reply:
In this case, I think it's a bit cleaner to add `SQLException` to the `throws` list since the functionality doesn't change and it removes some code and manual handling.